### DISCUSSION TITLE
Show parents of pages in the results of tag search.

### DISCUSSION THREAD

**Discussion by srjskam** [zim-desktop-wiki/zim-desktop-wiki#2403] [Ideas]
Using Journal + Tags plugins.

If I add tags to my journal entries and later search by tag in the Tags tab, the resulting list is rather confusing, listing days from 1 to 31. It would be useful to **see the whole path** (or n parents) in the search results, possibly optionally. The resulting list could be displayed as is, or maybe aligned right, clipping overflowing parents with "..." etc.

I imagine this use case might be pretty common, I assume both these plugins are popular. The same problem might arise if a user organizes their notes in a way that produces identical or similarly named pages, like "Summary", or "Chapter 1" etc.

Here's a picture of the problem. I've had a dream (tagged "uni") on the fifth of _some_ month of _some_ year.

![image](https://github.com/zim-desktop-wiki/zim-desktop-wiki/assets/26146452/220b1e9e-c5c8-485a-aef4-12bcfe7895af)

**Comment by introt** [comment]
You can right-click the page names and select "Show full page name"

![image](https://github.com/zim-desktop-wiki/zim-desktop-wiki/assets/26069787/9e252d71-05b0-4ba1-af71-c27112bffe8a)

This isn't covered in the tag plugin's manual which is a documentation issue.

**Reply by srjskam** [reply]
Thank you, dumb of me not to have found that. This is sufficient for my use, but it would be better if you could also sort the list by the whole path. Currently it is sorted only by the page.

**Reply by introt** [reply]
No worries. I wouldn't blame myself for not discovering undocumented features. There's an open issue for the sorting: #1632
